### PR TITLE
Set .conf parameters to reasonable default or empty else

### PR DESCRIPTION
**Pull request by mstormi**
Fixes: #1461 

Signed-off-by: Markus Storm <markus.storm@gmx.net>

### CODE REVIEW

File: build-image/openhabian.pi-raspios32.conf [hunk 1]
```diff
@@ -47,16 +47,16 @@ debugmode=off
...
 # force treating your box as if it was a ...
-# pi4, pi4_8gb, cm4, pi400, pi3, cm3, pi3+, cm3+, pi2, pi1, cm1, pi0, pi0w
-#hw=pi4
+# pi4, pi4_8gb, cm4, pi400, pi3, cm3, pi3+, cm3+, pi2, pi1, cm1, pi0, pi0w, x86
+# hw=
 
 # Hardware architecture
 # x86_64, amd64, armv6l, armv7l, aarch64, arm64
-#hwarch=armv7l
+hwarch=armv71
```
Comment (ecdye) [inline build-image/openhabian.pi-raspios32.conf:55]: Do we really want to set by default, why not just leave them as an optional override?

File: build-image/openhabian.pi-raspios64beta.conf [hunk 2]
```diff
@@ -47,16 +47,16 @@ debugmode=off
...
 # force treating your box as if it was a ...
-# pi4, pi4_8gb, cm4, pi400, pi3, cm3, pi3+, cm3+, pi2, pi1, cm1, pi0, pi0w
-#hw=pi4
+# pi4, pi4_8gb, cm4, pi400, pi3, cm3, pi3+, cm3+, pi2, pi1, cm1, pi0, pi0w, x86
+hw=pi4
 
 # Hardware architecture
 # x86_64, amd64, armv6l, armv7l, aarch64, arm64
-#hwarch=armv7l
+hwarch=aarch64
```
Comment (ecdye) [inline build-image/openhabian.pi-raspios64beta.conf:55]: See above.

**Review by ecdye — CHANGES_REQUESTED** [review]

File: build-image/openhabian.pi-raspios32.conf (hunk 1 above)
Comment (mstormi) [reply build-image/openhabian.pi-raspios32.conf:55]: I was wondering too. But what for do we have those -raspios32[64].conf files if not to make these differences?

Comment (ecdye) [reply build-image/openhabian.pi-raspios32.conf:55]: The Java differences at build time. We could probably consolidate them, and I can look into it. Not sure if it is necessary.

Comment (mstormi) [reply build-image/openhabian.pi-raspios32.conf]: ok now? then pls merge

**Review by ecdye — APPROVED** [review]
LGTM

**Pull request merged**